### DISCUSSION TITLE
Some confusion about rolldown/test

### DISCUSSION THREAD

**Discussion by moushicheng** [rolldown/rolldown#673] [General]
when i use
```
 npm run test
```
only create array/dist folder
<img width="872" alt="image" src="https://github.com/rolldown/rolldown/assets/54257412/efa5d3d3-5e37-40dc-a855-f8148ca49cc8">
why other case cant create their dist?

packages/rolldown/test/runner.ts tell me every case will write their dist

code is here⬇️
```
async function runCaseBundle(caseRoot: string, config?: RollupOptions) {
  config = normalizedOptions(caseRoot, config)
  const build = await rolldown(config as InputOptions)
  return await build.write(config.output as OutputOptions)
}
```

**Comment by hyfdev** [comment]
Maybe try latest main branch. I just tested locally. It worked fine.

**Reply by moushicheng** [reply]
yeah, i seem solve this problem in latest
